internal/dependencyinjection/event_manager.go

```diff
@@ -0,0 +1,30 @@
+package dependencyinjection
```

Comment:
Yeah, we need this because of the `TestServe` since it will be the same object, the test will pass. Otherwise, they will have different memory addresses, and the test fails.

Reply:
changed: [9ee7d22](https://github.com/stellar/stellar-disbursement-platform-backend/pull/119/commits/9ee7d22a1d0d32495233dfe75be43cc790dba533)